packages/apps/plugins/plugin-debug/src/components/DebugPanel.tsx

```diff
@@ -35,6 +39,18 @@ export const DebugPanel: FC<PropsWithChildren<{ menu: ReactNode }>> = ({ menu, c
           </div>
         )}
       </div>
+    </MainOrArticle>
+  );
+};
+
+const MainOrArticle: FC<PropsWithChildren<{ role: 'main' | 'article' }>> = ({ role, children }) => {
+  return role === 'main' ? (
+    <Main.Content classNames={[baseSurface, fixedInsetFlexLayout, topbarBlockPaddingStart, bottombarBlockPaddingEnd]}>
+      {children}
     </Main.Content>
+  ) : (
+    <div role='none' className='row-span-2 rounded-t-md overflow-x-auto'>
+      {children}
+    </div>
   );
```

Comment:
Better, I think, for `DebugPanel` to implement everything both `main` and `article` have in common, then just wrap them differently in the `provides.components` perhaps.

Creating new interceding components like this can cause issues if e.g. we need to start forwarding refs — instead of calling `forwardRef` multiple times because of the interceding component here, you could just pass the ref directly to `DebugPanel`.

Not blocking, just something to consider.